Remote Controller support
Enable support for Remote Controllers on Linux. This is
needed in order to support several video capture adapters,
standalone IR receivers/transmitters, and RF receivers.

Enable this option if you have a video capture board even
if you don't need IR, as otherwise, you may not be able to
compile the driver for your adapter.

Say Y when you have a TV or an IR device.
Depends: INPUT
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:19:54.568610339 -0400
@@ -4599,6 +4599,7 @@
 # PCI GPIO expanders
 #
 # CONFIG_GPIO_AMD8111 is not set
+# CONFIG_GPIO_BT8XX is not set
 # CONFIG_GPIO_ML_IOH is not set
 CONFIG_GPIO_PCI_IDIO_16=m
 # CONFIG_GPIO_PCIE_IDIO_24 is not set
@@ -5264,42 +5265,7 @@
 # CONFIG_REGULATOR_TPS65132 is not set
 # CONFIG_REGULATOR_TPS6524X is not set
 CONFIG_REGULATOR_TPS68470=m
-CONFIG_RC_CORE=y
-CONFIG_BPF_LIRC_MODE2=y
-CONFIG_LIRC=y
-CONFIG_RC_MAP=m
-CONFIG_RC_DECODERS=y
-CONFIG_IR_IMON_DECODER=m
-CONFIG_IR_JVC_DECODER=m
-CONFIG_IR_MCE_KBD_DECODER=m
-CONFIG_IR_NEC_DECODER=m
-CONFIG_IR_RC5_DECODER=m
-CONFIG_IR_RC6_DECODER=m
-CONFIG_IR_RCMM_DECODER=m
-CONFIG_IR_SANYO_DECODER=m
-CONFIG_IR_SHARP_DECODER=m
-CONFIG_IR_SONY_DECODER=m
-CONFIG_IR_XMP_DECODER=m
-CONFIG_RC_DEVICES=y
-CONFIG_IR_ENE=m
-CONFIG_IR_FINTEK=m
-CONFIG_IR_IGORPLUGUSB=m
-CONFIG_IR_IGUANA=m
-CONFIG_IR_IMON=m
-CONFIG_IR_IMON_RAW=m
-CONFIG_IR_ITE_CIR=m
-CONFIG_IR_MCEUSB=m
-CONFIG_IR_NUVOTON=m
-CONFIG_IR_REDRAT3=m
-CONFIG_IR_SERIAL=m
-CONFIG_IR_SERIAL_TRANSMITTER=y
-CONFIG_IR_STREAMZAP=m
-CONFIG_IR_TOY=m
-CONFIG_IR_TTUSBIR=m
-CONFIG_IR_WINBOND_CIR=m
-CONFIG_RC_ATI_REMOTE=m
-CONFIG_RC_LOOPBACK=m
-CONFIG_RC_XBOX_DVD=m
+# CONFIG_RC_CORE is not set
 CONFIG_CEC_CORE=m
 CONFIG_CEC_NOTIFIER=y
 CONFIG_CEC_PIN=y
@@ -5307,14 +5273,12 @@
 #
 # CEC support
 #
-CONFIG_MEDIA_CEC_RC=y
 # CONFIG_CEC_PIN_ERROR_INJ is not set
 CONFIG_MEDIA_CEC_SUPPORT=y
 CONFIG_CEC_CH7322=m
 # CONFIG_CEC_CROS_EC is not set
 CONFIG_CEC_GPIO=m
 CONFIG_CEC_SECO=m
-CONFIG_CEC_SECO_RC=y
 CONFIG_USB_PULSE8_CEC=m
 CONFIG_USB_RAINSHADOW_CEC=m
 # end of CEC support
@@ -5469,14 +5433,9 @@
 #
 CONFIG_VIDEO_AU0828=m
 CONFIG_VIDEO_AU0828_V4L2=y
-# CONFIG_VIDEO_AU0828_RC is not set
 CONFIG_VIDEO_CX231XX=m
-CONFIG_VIDEO_CX231XX_RC=y
 CONFIG_VIDEO_CX231XX_ALSA=m
 CONFIG_VIDEO_CX231XX_DVB=m
-CONFIG_VIDEO_TM6000=m
-CONFIG_VIDEO_TM6000_ALSA=m
-CONFIG_VIDEO_TM6000_DVB=m
 
 #
 # Digital TV USB devices
@@ -5494,38 +5453,9 @@
 CONFIG_DVB_USB_DVBSKY=m
 CONFIG_DVB_USB_EC168=m
 CONFIG_DVB_USB_GL861=m
-CONFIG_DVB_USB_LME2510=m
 CONFIG_DVB_USB_MXL111SF=m
 CONFIG_DVB_USB_RTL28XXU=m
 CONFIG_DVB_USB_ZD1301=m
-CONFIG_DVB_USB=m
-# CONFIG_DVB_USB_DEBUG is not set
-CONFIG_DVB_USB_A800=m
-CONFIG_DVB_USB_AF9005=m
-CONFIG_DVB_USB_AF9005_REMOTE=m
-CONFIG_DVB_USB_AZ6027=m
-CONFIG_DVB_USB_CINERGY_T2=m
-CONFIG_DVB_USB_CXUSB=m
-CONFIG_DVB_USB_CXUSB_ANALOG=y
-CONFIG_DVB_USB_DIB0700=m
-CONFIG_DVB_USB_DIB3000MC=m
-CONFIG_DVB_USB_DIBUSB_MB=m
-# CONFIG_DVB_USB_DIBUSB_MB_FAULTY is not set
-CONFIG_DVB_USB_DIBUSB_MC=m
-CONFIG_DVB_USB_DIGITV=m
-CONFIG_DVB_USB_DTT200U=m
-CONFIG_DVB_USB_DTV5100=m
-CONFIG_DVB_USB_DW2102=m
-CONFIG_DVB_USB_GP8PSK=m
-CONFIG_DVB_USB_M920X=m
-CONFIG_DVB_USB_NOVA_T_USB2=m
-CONFIG_DVB_USB_OPERA1=m
-CONFIG_DVB_USB_PCTV452E=m
-CONFIG_DVB_USB_TECHNISAT_USB2=m
-CONFIG_DVB_USB_TTUSB2=m
-CONFIG_DVB_USB_UMT_010=m
-CONFIG_DVB_USB_VP702X=m
-CONFIG_DVB_USB_VP7045=m
 CONFIG_SMS_USB_DRV=m
 CONFIG_DVB_TTUSB_BUDGET=m
 CONFIG_DVB_TTUSB_DEC=m
@@ -5537,7 +5467,6 @@
 CONFIG_VIDEO_EM28XX_V4L2=m
 CONFIG_VIDEO_EM28XX_ALSA=m
 CONFIG_VIDEO_EM28XX_DVB=m
-CONFIG_VIDEO_EM28XX_RC=m
 CONFIG_MEDIA_PCI_SUPPORT=y
 
 #
@@ -5553,10 +5482,6 @@
 # Media capture/analog TV support
 #
 # CONFIG_VIDEO_DT3155 is not set
-CONFIG_VIDEO_IVTV=m
-# CONFIG_VIDEO_IVTV_ALSA is not set
-CONFIG_VIDEO_FB_IVTV=m
-# CONFIG_VIDEO_FB_IVTV_FORCE_PAT is not set
 CONFIG_VIDEO_HEXIUM_GEMINI=m
 CONFIG_VIDEO_HEXIUM_ORION=m
 CONFIG_VIDEO_MXB=m
@@ -5564,23 +5489,9 @@
 #
 # Media capture/analog/hybrid TV support
 #
-CONFIG_VIDEO_BT848=m
-CONFIG_DVB_BT8XX=m
-CONFIG_VIDEO_CX18=m
-CONFIG_VIDEO_CX18_ALSA=m
-CONFIG_VIDEO_CX23885=m
-CONFIG_MEDIA_ALTERA_CI=m
 # CONFIG_VIDEO_CX25821 is not set
-CONFIG_VIDEO_CX88=m
-CONFIG_VIDEO_CX88_ALSA=m
-CONFIG_VIDEO_CX88_BLACKBIRD=m
-CONFIG_VIDEO_CX88_DVB=m
-CONFIG_VIDEO_CX88_ENABLE_VP3054=y
-CONFIG_VIDEO_CX88_VP3054=m
-CONFIG_VIDEO_CX88_MPEG=m
 CONFIG_VIDEO_SAA7134=m
 CONFIG_VIDEO_SAA7134_ALSA=m
-CONFIG_VIDEO_SAA7134_RC=y
 CONFIG_VIDEO_SAA7134_DVB=m
 CONFIG_VIDEO_SAA7134_GO7007=m
 CONFIG_VIDEO_SAA7164=m
@@ -5592,19 +5503,13 @@
 # CONFIG_DVB_B2C2_FLEXCOP_PCI_DEBUG is not set
 CONFIG_DVB_DDBRIDGE=m
 # CONFIG_DVB_DDBRIDGE_MSIENABLE is not set
-CONFIG_DVB_DM1105=m
-CONFIG_MANTIS_CORE=m
-CONFIG_DVB_MANTIS=m
-CONFIG_DVB_HOPPER=m
 CONFIG_DVB_NETUP_UNIDVB=m
 CONFIG_DVB_NGENE=m
 CONFIG_DVB_PLUTO2=m
 CONFIG_DVB_PT1=m
 # CONFIG_DVB_PT3 is not set
-CONFIG_DVB_SMIPCIE=m
 CONFIG_DVB_BUDGET_CORE=m
 CONFIG_DVB_BUDGET=m
-CONFIG_DVB_BUDGET_CI=m
 CONFIG_DVB_BUDGET_AV=m
 CONFIG_VIDEO_IPU3_CIO2=m
 CONFIG_CIO2_BRIDGE=y
@@ -5744,7 +5649,6 @@
 CONFIG_VIDEO_SAA7146=m
 CONFIG_VIDEO_SAA7146_VV=m
 CONFIG_SMS_SIANO_MDTV=m
-CONFIG_SMS_SIANO_RC=y
 # CONFIG_SMS_SIANO_DEBUGFS is not set
 CONFIG_VIDEOBUF2_CORE=m
 CONFIG_VIDEOBUF2_V4L2=m
@@ -5763,11 +5667,6 @@
 CONFIG_MEDIA_ATTACH=y
 
 #
-# IR I2C driver auto-selected by 'Autoselect ancillary drivers'
-#
-CONFIG_VIDEO_IR_I2C=m
-
-#
 # Camera sensor devices
 #
 CONFIG_VIDEO_APTINA_PLL=m
@@ -5857,19 +5756,13 @@
 #
 # audio, video and radio I2C drivers auto-selected by 'Autoselect ancillary drivers'
 #
-CONFIG_VIDEO_CS3308=m
-CONFIG_VIDEO_CS5345=m
 CONFIG_VIDEO_CS53L32A=m
 CONFIG_VIDEO_MSP3400=m
 CONFIG_VIDEO_SONY_BTF_MPX=m
-CONFIG_VIDEO_TDA7432=m
 CONFIG_VIDEO_TDA9840=m
 CONFIG_VIDEO_TEA6415C=m
 CONFIG_VIDEO_TEA6420=m
-CONFIG_VIDEO_TVAUDIO=m
 CONFIG_VIDEO_UDA1342=m
-CONFIG_VIDEO_VP27SMPX=m
-CONFIG_VIDEO_WM8739=m
 CONFIG_VIDEO_WM8775=m
 CONFIG_VIDEO_SAA6588=m
 CONFIG_VIDEO_SAA711X=m
@@ -5881,13 +5774,8 @@
 #
 # Video and audio decoders
 #
-CONFIG_VIDEO_SAA717X=m
 CONFIG_VIDEO_CX25840=m
-CONFIG_VIDEO_SAA7127=m
-CONFIG_VIDEO_UPD64031A=m
-CONFIG_VIDEO_UPD64083=m
 CONFIG_VIDEO_SAA6752HS=m
-CONFIG_VIDEO_M52790=m
 
 #
 # SPI I2C drivers auto-selected by 'Autoselect ancillary drivers'
@@ -5911,14 +5799,11 @@
 CONFIG_MEDIA_TUNER_FC0013=m
 CONFIG_MEDIA_TUNER_FC2580=m
 CONFIG_MEDIA_TUNER_IT913X=m
-CONFIG_MEDIA_TUNER_M88RS6000T=m
-CONFIG_MEDIA_TUNER_MAX2165=m
 CONFIG_MEDIA_TUNER_MC44S803=m
 CONFIG_MEDIA_TUNER_MT2060=m
 CONFIG_MEDIA_TUNER_MT2063=m
 CONFIG_MEDIA_TUNER_MT20XX=m
 CONFIG_MEDIA_TUNER_MT2131=m
-CONFIG_MEDIA_TUNER_MT2266=m
 CONFIG_MEDIA_TUNER_MXL5005S=m
 CONFIG_MEDIA_TUNER_MXL5007T=m
 CONFIG_MEDIA_TUNER_QM1D1B0004=m
@@ -5929,7 +5814,6 @@
 CONFIG_MEDIA_TUNER_SIMPLE=m
 CONFIG_MEDIA_TUNER_TDA18212=m
 CONFIG_MEDIA_TUNER_TDA18218=m
-CONFIG_MEDIA_TUNER_TDA18250=m
 CONFIG_MEDIA_TUNER_TDA18271=m
 CONFIG_MEDIA_TUNER_TDA827X=m
 CONFIG_MEDIA_TUNER_TDA8290=m
@@ -5951,7 +5835,6 @@
 CONFIG_DVB_M88DS3103=m
 CONFIG_DVB_MXL5XX=m
 CONFIG_DVB_STB0899=m
-CONFIG_DVB_STB6100=m
 CONFIG_DVB_STV090x=m
 CONFIG_DVB_STV0910=m
 CONFIG_DVB_STV6110x=m
@@ -5969,18 +5852,11 @@
 #
 # DVB-S (satellite) frontends
 #
-CONFIG_DVB_CX24110=m
 CONFIG_DVB_CX24116=m
-CONFIG_DVB_CX24117=m
 CONFIG_DVB_CX24120=m
 CONFIG_DVB_CX24123=m
-CONFIG_DVB_DS3000=m
-CONFIG_DVB_MB86A16=m
 CONFIG_DVB_MT312=m
 CONFIG_DVB_S5H1420=m
-CONFIG_DVB_SI21XX=m
-CONFIG_DVB_STB6000=m
-CONFIG_DVB_STV0288=m
 CONFIG_DVB_STV0299=m
 CONFIG_DVB_STV0900=m
 CONFIG_DVB_STV6110=m
@@ -6003,23 +5879,15 @@
 CONFIG_DVB_AF9013=m
 CONFIG_DVB_AS102_FE=m
 CONFIG_DVB_CX22700=m
-CONFIG_DVB_CX22702=m
 CONFIG_DVB_CXD2820R=m
 CONFIG_DVB_CXD2841ER=m
-CONFIG_DVB_DIB3000MB=m
-CONFIG_DVB_DIB3000MC=m
-CONFIG_DVB_DIB7000M=m
-CONFIG_DVB_DIB7000P=m
 CONFIG_DVB_DRXD=m
 CONFIG_DVB_EC100=m
-CONFIG_DVB_GP8PSK_FE=m
 CONFIG_DVB_L64781=m
 CONFIG_DVB_MT352=m
-CONFIG_DVB_NXT6000=m
 CONFIG_DVB_RTL2830=m
 CONFIG_DVB_RTL2832=m
 CONFIG_DVB_SI2168=m
-CONFIG_DVB_SP887X=m
 CONFIG_DVB_STV0367=m
 CONFIG_DVB_TDA10048=m
 CONFIG_DVB_TDA1004X=m
@@ -6047,15 +5915,12 @@
 CONFIG_DVB_LGDT330X=m
 CONFIG_DVB_MXL692=m
 CONFIG_DVB_NXT200X=m
-CONFIG_DVB_OR51132=m
-CONFIG_DVB_OR51211=m
 CONFIG_DVB_S5H1409=m
 CONFIG_DVB_S5H1411=m
 
 #
 # ISDB-T (terrestrial) frontends
 #
-CONFIG_DVB_DIB8000=m
 CONFIG_DVB_MB86A20S=m
 CONFIG_DVB_S921=m
 
@@ -6068,8 +5933,6 @@
 # Digital terrestrial only tuners/PLL
 #
 CONFIG_DVB_PLL=m
-CONFIG_DVB_TUNER_DIB0070=m
-CONFIG_DVB_TUNER_DIB0090=m
 
 #
 # SEC control devices for DVB-S
@@ -6077,19 +5940,13 @@
 CONFIG_DVB_A8293=m
 CONFIG_DVB_AF9033=m
 CONFIG_DVB_ASCOT2E=m
-CONFIG_DVB_ATBM8830=m
 CONFIG_DVB_HELENE=m
 CONFIG_DVB_HORUS3A=m
 CONFIG_DVB_ISL6405=m
 CONFIG_DVB_ISL6421=m
 CONFIG_DVB_ISL6423=m
-CONFIG_DVB_IX2505V=m
-CONFIG_DVB_LGS8GXX=m
 CONFIG_DVB_LNBH25=m
 CONFIG_DVB_LNBP21=m
-CONFIG_DVB_LNBP22=m
-CONFIG_DVB_M88RS2000=m
-CONFIG_DVB_TDA665x=m
 CONFIG_DVB_DRX39XYJ=m
 
 #
@@ -7090,7 +6947,6 @@
 CONFIG_HID_PICOLCD_BACKLIGHT=y
 CONFIG_HID_PICOLCD_LCD=y
 CONFIG_HID_PICOLCD_LEDS=y
-CONFIG_HID_PICOLCD_CIR=y
 CONFIG_HID_PLANTRONICS=m
 CONFIG_HID_PLAYSTATION=m
 CONFIG_PLAYSTATION_FF=y
